File: 2862.c
#include <stdio.h>

int main() {
  int c, n;

  while (scanf("%d", &c) != EOF) {
    for (int i = 0; i < c; i++) {
      scanf("%d", &n);

      if (n > 8000) {
        printf("Mais de 8000!\n");
      } else {
        printf("Inseto!\n");
      }
    }
  }
  return 0;
}

File: 2862.py
# Python 3.4.3

def main():
  while True:
    try:
      for i in range(int(input())):
        if(int(input())) > 8000:
          print('Mais de 8000!')
        else:
          print('Inseto!')
    except EOFError:
      break

if __name__ == '__main__':
	main()